src/cocoa/toga_cocoa/libs/appkit.py

```diff
@@ -67,11 +67,22 @@ class NSAlertStyle(Enum):
 NSApplicationDidHideNotification = c_void_p.in_dll(appkit, 'NSApplicationDidHideNotification')
 NSApplicationDidUnhideNotification = c_void_p.in_dll(appkit, 'NSApplicationDidUnhideNotification')
 
-NSAboutPanelOptionApplicationIcon = NSString(c_void_p.in_dll(appkit, "NSAboutPanelOptionApplicationIcon"))
-NSAboutPanelOptionApplicationName = NSString(c_void_p.in_dll(appkit, "NSAboutPanelOptionApplicationName"))
-NSAboutPanelOptionApplicationVersion = NSString(c_void_p.in_dll(appkit, "NSAboutPanelOptionApplicationVersion"))
-NSAboutPanelOptionCredits = NSString(c_void_p.in_dll(appkit, "NSAboutPanelOptionCredits"))
-NSAboutPanelOptionVersion = NSString(c_void_p.in_dll(appkit, "NSAboutPanelOptionVersion"))
+# NSAboutPanelOption* keys are available only 10.13+
+platform_version, _, _ = platform.mac_ver()
+platform_version = float('.'.join(platform_version.split('.')[:2]))
+
+if platform_version >= 10.13:
```

Comment:
I get the intent here, and this will definitely work - but honestly, it's probably better to "ask forgiveness rather than permission", and catch the Attribute Error when the attribute doesn't exist, rather than parse an explicit version number. It will be faster in both the success and failure cases, and less prone to error (since there is literally no logic to go wrong.)